src/components/DisplayItems/index.tsx

```diff
@@ -0,0 +1,59 @@
+import Heading from "@theme/Heading"
+
+import React from "react"
+import Link from "@docusaurus/Link"
+import clsx from "clsx"
+
+import styles from "./styles.module.css"
+
+export type Item = {
+	title: string
+	video: string
+	description: string
+}
+
+export function DisplayItemsHeader({
+	heading,
+	description,
+	cta,
```

Comment:
What's the `cta` meaning?

Reply:
Cta = call to action, bit of a stretch - I'll rename it.